tests/phpunit/CRM/Utils/Mail/EmailProcessorTest.php

```diff
@@ -105,4 +129,27 @@ public function setUpMailing() {
     $this->eventQueue = $this->callAPISuccess('MailingEventQueue', 'get', array('api.MailingEventQueue.create' => array('hash' => 'aaaaaaaaaaaaaaaa')));
   }
 
+  /**
+   * Remove directory, including files within it.
+   *
+   * @param string $dir
+   */
+  protected function rrmdir($dir) {
```

Comment:
There's a utility function `CRM_Utils_File::cleanDir($dir, TRUE, FALSE)` which should be pretty similar. I'm not sure if there's some intentional/significant difference here -- but, FWIW, I've used it in other tests (like `CRM_Extension_Manager_ModuleTest`).

Reply:
@totten yep - that function worked